test/runtests.jl

```diff
@@ -15,6 +15,29 @@ h1[4,3:4] .= 4.5
 h1[5,4] = 5.5
 h1[5,2] = 6.5
 
+@testset "SimpleHypergraphs building models    " begin
+
+    Hᵣ = random_model(5,5)
+    @test nhv(Hᵣ) == 5
+    @test nhe(Hᵣ) == 5
+    @test  all(length.(Hᵣ.v2he) .> 0)
+    @test  all(length.(Hᵣ.v2he) .<= 5)
+
```

Comment:
Add tests for corner cases with `0` and `1` values of one parameter and normal values of the other.